index.bs

```diff
@@ -225,27 +225,11 @@ The following steps are taken:
 1. If Type(_loader_) is not Object, throw a *TypeError* exception.
 1. If _stage_ is *undefined* then let _stage_ be "ready".
 1. Return the result of transforming Resolve(_loader_, _name_, _referrer_) with a fulfillment handler that, when called with argument _key_, runs the following steps:
-  1. If _stage_ is "fetch", then:
-    1. Return the result of transforming RequestFetch(_loader_, _key_) with a new pass-through promise.
-  1. If _stage_ is "translate", then:
-    1. Return the result of transforming RequestTranslate(_loader_, _key_) with a new pass-through promise.
-  1. If _stage_ is "instantiate", then:
-    1. Return the result of transforming RequestInstantiate(_loader_, _key_) with a fulfillment handler that, when called with argument _entry_, runs the following steps:
-      1. If _entry_.[[Module]] is a Function object, return _entry_.[[Module]].
-      1. Return *undefined*.
-  1. If _stage_ is "satisfy", then:
-    1. Return the result of transforming RequestSatisfy(_loader_, _key_) with a fulfillment handler that, when called with argument _entry_, runs the following steps:
-      1. If _entry_.[[Module]] is a Function object, return _entry_.[[Module]].
-      1. Return *undefined*.
-  1. If _stage_ is "link", then:
-    1. Return the result of transforming RequestLink(_loader_, _key_) with a fulfillment handler that returns *undefined*.
-  1. If _stage_ is "ready", then:
-    1. Return the result of transforming RequestReady(_loader_, _key_) with a fulfillment handler that, when called with argument _entry_, runs the following steps:
-      1. Return GetModuleNamespace(_entry_.[[Module]]).
-  1. Throw a new *TypeError*.
+  1. Let _entry_ be EnsureRegistered(_loader_, _key_).
+  1. Return _entry_.load(_stage_).
```

Comment:
I don't think we want to call the public .load method on the entry, do we? Don't we want to call an abstract operation (so it can't be overwritten in userland)?